docs/guides/deploy-k8s.md

```diff
@@ -27,6 +27,118 @@ This guide explains how to deploy the Sensitive Data Archive (SDA) in kubernetes
 
 ## Minimal working configuration
 
+## Minimal set of configuration variables
+
+Here we provide a minimal list of variables that need to be configured in the `values.yml` file of the Helm charts in order to achieve a working deployment of the `sensitive data archive` services. In the following it is assumed that a *federated* setup is being deployed.
+
+## Global Variables
+
+TLS support:
+- `global.tls.issuer` or `glbal.tls.clusterIssuer`
+
+Storage:
+- `global.archive.storageType`: The storage type for the archive. If this is set to `s3`, then the following variables need to be set as well:
+- `global.archive.s3Url`: The S3 URL for the archive.
+- `global.archive.s3Bucket`: The S3 bucket for the archive.
+- `global.archive.s3AccessKey`: The S3 access key for the archive.
+- `global.archive.s3SecretKey`: The S3 secret key for the archive.
+
+- `global.backupArchive.storageType`: The storage type for the backup archive. If this is set to `s3`, then the following variables need to be set as well:
+- `global.backupArchive.s3Url`: The S3 URL for the backup archive.
+- `global.backupArchive.s3Bucket`: The S3 bucket for the backup archive.
+- `global.backupArchive.s3AccessKey`: The S3 access key for the backup archive.
+- `global.backupArchive.s3SecretKey`: The S3 secret key for the backup archive.
+
+- `global.inbox.storageType`: The storage type for the inbox. If this is set to `s3`, then the following variables need to be set as well:
+- `global.inbox.s3Url`: The S3 URL for the inbox.
+- `global.inbox.s3Bucket`: The S3 bucket for the inbox.
+- `global.inbox.s3AccessKey`: The S3 access key for the inbox.
+- `global.inbox.s3SecretKey`: The S3 secret key for the inbox.
+
+RabbitMQ:
+- `global.broker.host`: The host for the broker.
+- `global.broker.exchange`: The exchange for the broker.
+- `global.broker.routingError`: The routing error for the broker.
+- `global.broker.backupRoutingKey`: The backup routing key for the broker.
+
+Crypt4gh:
+- `global.c4gh.secretName`: The secret name for c4gh.
```

Comment:
maybe to specify that this is a k8s secret and it is the name by which it is referenced in the chart